app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,150 @@
+<div>
+<% if @recipe.public || @recipe.user == current_user %>
+<h1 class="show-recipe-heading">Recipe <%= Recipe.all.index(@recipe) + 1 %></h1>
+    <h2 class="show-recipe-name"><%= @recipe.name %></h2>
+
+    <div id="recipe-details-information">
+    <div class="prepare-toggle">
+      <p>Preparation time: <b><%= @recipe.preparation_time_hours %> hours</b></p>
+
+      <div>
+
+      <% if @recipe.user == current_user %>
+  <div class="toggle-container">
+    <%= form_for @recipe, url: recipe_path(@recipe), method: :patch do |f| %>
+      <label for="toggle">
+        <b>Public</b>
+      </label>
+      <div class="toggle-label">
+        <%= f.check_box :public, id: 'toggle', class: 'toggle-input' %>
+        <%= f.label :public, class: 'toggle-label', for: 'toggle' do %>
+          <div class="toggle-indicator"></div>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+<% end %>
+      
+        
+      </div>
+    </div>
+    <p>Cooking time: <b><%= @recipe.cooking_time_hours %> hours</b></p>
+    <hr style="margin-top: 10px; margin-bottom: 10px;">
+    <p class="fs-5"><%= @recipe.description %></p>
+  </div>
+
+
+  <div id="shopping-list-generate-button">
+    <button type="button"
+    id="generate-shopping-list"
+      class="btn btn-primary"
+      data-bs-toggle="modal"
+      data-bs-target="#myModal"
+      onclick="showModal()">
+        Generate shopping list
+    </button>
+
+    
```

Comment:
- [ ] As required for the project, kindly ensure that user can create a shopping list as in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food_shopping_list.png?raw=true). Right now, your application doesn't accept create a shopping list as it always displays the alert message after clicking the **Generate** button.
![ksnip_20231123-184335](https://github.com/MenendezON/Recipe-app/assets/83042742/c56695ab-076a-4d71-842a-0048409d869b)